docs/community/contribute.md

```diff
@@ -27,45 +27,157 @@ Quick reference:
 git clone https://github.com/<your-username>/ragna.git
 ```
 
-### Set up development environment
+### Set up development environment and run Ragna
 
-We recommend using conda, but you can use any package and environment manager you
-prefer. The `environment-dev.yml` file at the root of the project lists all the required
-development dependencies.
+We use [Pixi](https://pixi.sh/dev/) to manage development environments with Ragna. The
+required development dependencies are listed in the `pyproject.toml` file at the root
```

Comment:
No need to call out the dev requirements anymore as pixi automatically handles this now.